pages/index.js

```diff
@@ -34,11 +34,11 @@ export default function Index({ redirectOrigin }) {
       <Header>
         <Heading.H1>Welcome!</Heading.H1>
         <Text>
-          There is litle understanding about mental disabilities, which is
+          There is only a little understanding about mental disabilities, which is
```

Comment:
```suggestion
          Most people have very little knowledge about mental disabilities, which is
```

Could this be rephrased like this? Maybe it doesn't work in this context.

Reply:
Most likely. I was a bit confused about the sentence and didn't know what it should say